runelite-client/src/main/java/net/runelite/client/plugins/randomevents/RandomEventConfig.java

```diff
@@ -44,129 +44,120 @@
 		keyName = "removeMenuOptions",
 		name = "Remove others' menu options",
 		description = "Remove menu options from random events for other players.",
-		position = 0
+		position = -3
 	)
 	default boolean removeMenuOptions()
 	{
 		return true;
 	}
 
 	@ConfigItem(
-		keyName = "notifyDunce",
-		name = "Notify on Surprise Exam",
-		description = "",
-		position = 1,
-		section = notificationSection
+			keyName = "notifyAll",
+			name = "Notify for all events",
+			description = "",
+			position = -1,
+			section = notificationSection
```

Comment:
All of the added/modified configitems are indented one level too far.
```suggestion
		keyName = "notifyAll",
		name = "Notify for all events",
		description = "",
		position = -1,
		section = notificationSection
```

Reply:
done